src/main/scala/beam/sim/RunBeam.scala

```diff
@@ -1,6 +1,11 @@
 package beam.sim
 
+import ch.qos.logback.classic.util.ContextInitializer
+
 object RunBeam extends BeamHelper {
+  val isLogbackConfigSet = Option(System.getProperty(ContextInitializer.CONFIG_FILE_PROPERTY)).isDefined
+  if (!isLogbackConfigSet)
```

Comment:
Suggestion:

Option 1:
```Scala
val logbackConfigFile = Option(System.getProperty(ContextInitializer.CONFIG_FILE_PROPERTY))
if (logbackConfigFile.isEmpty) {
    System.setProperty(ContextInitializer.CONFIG_FILE_PROPERTY, "logback.xml")
}
```

Option 2:  <<== I would choose this one since you are not using the content of the variable anywhere
```Scala
if (System.getProperty(ContextInitializer.CONFIG_FILE_PROPERTY) == null) {
    System.setProperty(ContextInitializer.CONFIG_FILE_PROPERTY, "logback.xml")
}
```